src/php/bin/run_all_docker_images.sh

```diff
@@ -16,7 +16,7 @@
 set -e
 cd $(dirname $0)/../../..
 
-ALL_IMAGES=( grpc-ext grpc-src alpine centos7 php5 php-src php-future php-zts
+ALL_IMAGES=( grpc-ext grpc-src alpine centos7 php-src php-future php-zts
```

Comment:
Can you also delete the dockerfile at `templates/src/php/docker/php5/Dockerfile.template` please?

Reply:
comment addressed

Reply:
Oh, and `src/php/docker/php5/Dockerfile` too please.

Reply:
comment addressed